code/modules/games/cards.dm

```diff
@@ -0,0 +1,469 @@
+/datum/playingcard
+	var/name = "playing card"
+	var/card_icon = "card_back"
+	var/back_icon = "card_back"
+
+/obj/item/weapon/deck
+	w_class = WEIGHT_CLASS_SMALL
+	icon = 'icons/obj/playing_cards.dmi'
+	var/list/cards = list()
+	var/cooldown = 0 // to prevent spam shuffle
+	throw_speed = 3
+	throw_range = 10
+	throwforce = 0
+	force = 0
+
+/obj/item/weapon/deck/holder
+	name = "card box"
+	desc = "A small leather case to show how classy you are compared to everyone else."
+	icon_state = "card_holder"
+
+/obj/item/weapon/deck/cards
+	name = "deck of cards"
+	desc = "A simple deck of playing cards."
+	icon_state = "deck"
+
+/obj/item/weapon/deck/cards/New()
+	..()
+	var/datum/playingcard/P
+	for(var/suit in list("spades","clubs","diamonds","hearts"))
+
+		var/colour
+		if(suit == "spades" || suit == "clubs")
+			colour = "black_"
+		else
+			colour = "red_"
+
+		for(var/number in list("ace","two","three","four","five","six","seven","eight","nine","ten"))
+			P = new()
+			P.name = "[number] of [suit]"
+			P.card_icon = "[colour]num"
+			P.back_icon = "card_back"
+			cards += P
+
+		for(var/number in list("jack","queen","king"))
+			P = new()
+			P.name = "[number] of [suit]"
+			P.card_icon = "[colour]col"
+			P.back_icon = "card_back"
+			cards += P
+
+	for(var/i = 0, i<2, i++)
+		P = new()
+		P.name = "joker"
+		P.card_icon = "joker"
+		cards += P
+
+/obj/item/weapon/deck/doublecards
+		name = "double deck of cards"
+		desc = "A simple deck of playing cards. Multiplied by two. Does not necessarily come with twice the fun."
+		icon_state = "doubledeck"
+
+/obj/item/weapon/deck/doublecards/New()
+	..()
+	var/datum/playingcard/P
+	for(var/suit in list("spades","clubs","diamonds","hearts"))
+
+		var/colour
+		if(suit == "spades" || suit == "clubs")
+			colour = "black_"
+		else
+			colour = "red_"
+
+		for(var/number in list("ace","two","three","four","five","six","seven","eight","nine","ten",))
+			P = new()
+			P.name = "[number] of [suit]"
+			P.card_icon = "[colour]num"
+			P.back_icon = "card_back"
+			cards += P
+
+		for(var/number in list("jack","queen","king"))
+			P = new()
+			P.name = "[number] of [suit]"
+			P.card_icon = "[colour]col"
+			P.back_icon = "card_back"
+			cards += P
+
+	for(var/i = 0, i<2, i++)
+		P = new()
+		P.name = "joker"
+		P.card_icon = "joker"
+		cards += P
+	cards += cards
```

Comment:
I'm pretty sure these will be a "shallow copy" of the items within `cards`. Not sure exactly WHAT weird stuff will happen as a result, but weird stuff is going to result.

You might just want to wrap the prior for loops in a `for i < 2` loop instead, to run the above twice.